040-Data-operations/160-image-transformations.mdx

```diff
@@ -351,11 +352,13 @@ https://us-east-1.storage.xata.sh/transform/download=xata-cat.jpg,format=jpeg/4u
 
 ### dpr
 
-This refers to the Device Pixel Ratio (DPR). The default is 1. This is a multiplier (similar to a flat scale) for width/height that makes it easier to specify higher-DPI sizes in `img srcset`. `dpr` needs to be used in tandem with a dimension transform to take effect.
+This refers to the Device Pixel Ratio (DPR). The default is 1. This is a multiplier (similar to a flat scale) for width/height that makes it easier to specify higher-DPI sizes in `img srcset`. `dpr` needs to be used in tandem with a (@snide, what is a dimension transform?) dimension transform to take effect.
```

Comment:
@snide what is a dimension transform?

Reply:
One that changes the height or width or both.

Reply:
I think this is fine